src/cpu/flexcpu/simple_dataflow_thread.hh

```diff
@@ -132,6 +132,7 @@ class SDCPUThread : public ThreadContext
     const bool inOrderExecute;
 
     const unsigned inflightInstsMaxSize;
```

Comment:
This would be better named "inflightOpsMaxSize" right? Maybe you should do that.

Reply:
I thought about it, but I figured it would be better to keep the name for the inflightInsts buffer, this variable, and the class name InflightInst consistent? I'm not sure if all the things should be changed :L

Reply:
Or if not all, which parts.

Reply:
Maybe you should change all of the names, actually.

Or keep it this way and make sure to add documentation to make it clear that "Inst" means `StaticInst`, which is a micro-op (maybe).

Reply:
Added documentation to the InflightInst header. I'm not sure if there is a cleaner solution, but I just hope that the documentation that's associated with the parameters themselves is sufficient.